tests/test_data_generators.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+
+from keras_eval.data_generators import AugmentedImageDataGenerator
+from keras_model_specs import ModelSpec
+
+
+def test_augmented_image_data_generator_wrong_scale_size(test_catdog_dataset_path):
+    model_spec = ModelSpec.get('test', preprocess_func='mean_subtraction',
+                               preprocess_args=[141., 130., 123.], target_size=(224, 224, 3))
+    with pytest.raises(ValueError) as exception:
+        test_data_generator = AugmentedImageDataGenerator(preprocessing_function=model_spec.preprocess_input,
+                                                          data_augmentation={'scale_sizes': 'asd'})
+        test_data_generator.flow_from_directory(
+            directory=test_catdog_dataset_path,
+            batch_size=1,
+            target_size=model_spec.target_size[:2],
+            class_mode='categorical',
+            shuffle=False)
+    expected = 'Incorrect format for `scale_sizes`, list of ints or `= default` is expected'
+    actual = str(exception).split('ValueError: ')[1]
+    assert actual == expected
+
+
+def test_augmented_image_data_generator_wrong_transforms(test_catdog_dataset_path):
+    model_spec = ModelSpec.get('test', preprocess_func='mean_subtraction',
+                               preprocess_args=[141., 130., 123.], target_size=(224, 224, 3))
+
+    with pytest.raises(ValueError) as exception:
+        test_data_generator = AugmentedImageDataGenerator(preprocessing_function=model_spec.preprocess_input,
+                                                          data_augmentation={'scale_sizes': [256],
+                                                                             'transforms': ['failure']})
+        datagen = test_data_generator.flow_from_directory(
+            directory=test_catdog_dataset_path,
+            batch_size=1,
+            target_size=model_spec.target_size[:2],
+            class_mode='categorical',
+            shuffle=False)
+        datagen.next()
+
+    expected = 'Wrong transform failure check documentation to see the supported ones'
```

Comment:
If you apply my proposed change from above (not a big deal), this should be now:
```suggestion
    expected = 'Wrong transform: failure . Check documentation to see the supported ones'
```